scripts/scripts.js

```diff
@@ -992,6 +991,21 @@ function getLinkLabel(element) {
   return element.title ? toClassName(element.title) : toClassName(element.textContent);
 }
 
+function findConversionValue(parent, fieldName) {
+  // Try to find the element by Id or Name
+  const valueElement = document.getElementById(fieldName) || parent.querySelector(`[name='${fieldName}']`);  
+  if (valueElement) {
+    return valueElement.value;
+  }
+  // Find the element by the inner text of the label
+  return Array.from(parent.getElementsByTagName('label'))
+    .filter(l => l.innerText === fieldName)
+    .map(label => document.getElementById(label.htmlFor))
+    .filter(field => field)
```

Comment:
```suggestion
    .filter(field => !!field)
```

makes it clearer that we are coercing to boolean here.

Reply:
Done